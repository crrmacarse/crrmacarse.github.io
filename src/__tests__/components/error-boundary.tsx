import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { create, ReactTestRenderer } from 'react-test-renderer';
import ErrorBoundary from 'providers/error-boundary';

let wrapper: ShallowWrapper;
let snapshot: ReactTestRenderer;

beforeEach(() => {
  wrapper = shallow(<ErrorBoundary />)
  snapshot = create(<ErrorBoundary />);
});

describe('<ErrorBoundary />', () => {
  it('matches the snapshot', () => {
    expect(snapshot.toJSON()).toMatchSnapshot();
  });

  it('renders', () => { wrapper });
});
