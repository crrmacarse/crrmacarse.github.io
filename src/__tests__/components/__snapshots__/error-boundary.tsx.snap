// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches the snapshot 1`] = `
<div
  className="not-found-section flex flex-col w-full h-screen justify-center items-center"
>
  <p
    className="text-6xl"
  >
    Page Not Found :(
  </p>
  <a
    className="my-3 font-semibold"
    href="/"
  >
    Return to the homepage.
  </a>
</div>
`;
